build.sbt

```diff
@@ -1,6 +1,6 @@
 import _root_.bintray.BintrayPlugin.bintrayPublishSettings
 
-val baseVersion = "6.0.0-SNAPSHOT"
+val baseVersion = "6.0.0-M1"
```

Comment:
let's not change this. I think it's handled by the release process and it will throw things off if it's manually changed